atd-vzv/src/views/dashboard/yearsOfLifeLost.js

```diff
@@ -11,15 +12,41 @@ const YearsOfLifeLost = () => {
   const lastYear = moment()
     .subtract(1, "year")
     .format("YYYY");
+  const twoYearsAgo = moment()
+    .subtract(2, "year")
+    .format("YYYY");
+  const threeYearsAgo = moment()
+    .subtract(3, "year")
+    .format("YYYY");
+  const fourYearsAgo = moment()
+    .subtract(4, "year")
+    .format("YYYY");
 
   const yearToDateUrl = `https://data.austintexas.gov/resource/xecs-rpy9.json?$where=prsn_injry_sev_id = '4' AND crash_date between '${thisYear}-01-01T00:00:00' and '${today}T23:59:59'`;
   const previousYearUrl = `https://data.austintexas.gov/resource/xecs-rpy9.json?$where=prsn_injry_sev_id = '4' AND crash_date between '${lastYear}-01-01T00:00:00' and '${lastYear}${todayMonthYear}T23:59:59'`;
+  const twoYearsAgoUrl = `https://data.austintexas.gov/resource/xecs-rpy9.json?$where=prsn_injry_sev_id = '4' AND crash_date between '${twoYearsAgo}-01-01T00:00:00' and '${twoYearsAgo}${todayMonthYear}T23:59:59'`;
+  const threeYearsAgoUrl = `https://data.austintexas.gov/resource/xecs-rpy9.json?$where=prsn_injry_sev_id = '4' AND crash_date between '${threeYearsAgo}-01-01T00:00:00' and '${threeYearsAgo}${todayMonthYear}T23:59:59'`;
+  const fourYearsAgoUrl = `https://data.austintexas.gov/resource/xecs-rpy9.json?$where=prsn_injry_sev_id = '4' AND crash_date between '${fourYearsAgo}-01-01T00:00:00' and '${fourYearsAgo}${todayMonthYear}T23:59:59'`;
 
```

Comment:
For lines 12-29, you could refactor this code to be less repetitive...

```js
const getFatailitiesByYearsAgoUrl = (yearsAgo) => {
   let yearsAgoDate = moment().subtract(yearsAgo, "year").format("YYYY");

   `https://data.austintexas.gov/resource/xecs-rpy9.json?$where=prsn_injry_sev_id = '4' AND crash_date between '${yearsAgoDate}-01-01T00:00:00' and '${yearsAgoDate}${todayMonthYear}T23:59:59'`;
}
```


Then below around line 75, for each of the axios calls, you could do something like...

```js
axios.get(getFatailitiesByYearsAgoUrl(2)).then(res => {
  setTwoYearsAgoYearToDateYearsLostTotal(getYearsOfLifeLost(res.data));
});
```